tests/recoverable_parser.rs

```diff
@@ -44,7 +47,9 @@ fn test_parse_invalid_block() -> Result<()> {
     )
     .unwrap();
     let config = ParserConfig::new().recover_block(true);
-    let (nodes, errors) = Parser::new(config).parse_recoverable(tokens).split_vec();
+    let (nodes, errors) = Parser::new(config)
+        .parse_recoverable::<Infallible>(tokens)
```

Comment:
Moving <C> to `ParserConfig` and `Parser` with default types, should return those changes back.